lib/types.ts

```diff
@@ -219,4 +219,12 @@ export type UploadResult = {
 };
 export interface BearerTokenPayload {
   formID?: string;
+  email?: string;
+}
+
+export interface FormUserDBConfigProperties {
```

Comment:
Typescript variables should be in camelCase. I know this doesn't line up with the naming convention in the database, so there will need to be a way to reconcile the differences, but I think the interface should be camelCase.

Reply:
I see but the entire project's interface follow this naming convention.  Maybe are you suggesting `FormUserDdConfigProperties` instead. Or am i missing something ?

Reply:
James in referring to `template_id` instead of `templateID` and `temporary_token` instead of `temporaryToken`